app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<link href="stylesheets/index.css" rel="stylesheet" type="text/css">
+        
+<head>
+  <title>Jillianne's TaskList</title>
+</head>
+<body>
+  <header>
+    <h1>Jillianne's TaskList</h1>
+  </header>
+  <section>
+      <% if @tasks.length == 0 %>
+      <h3>No Tasks</h3>
+      <% else %>
+          <% @tasks.each do |task| %>
+              <br>
+              <td><%= task.name %></td>
+              <% if task.completed_at.nil? %>
```

Comment:
You'll probably want to indent the code that's within the `if` (and also the `else`)